docs/en/sql-reference/sql-functions/table-functions/files.md

```diff
@@ -163,6 +163,8 @@ After the sampling, StarRocks unionizes the columns from all the data files acco
 - For columns with the same column name but different data types, they are identified as the same column but with a general data type on a relative fine granularity level. For example, if the column `col1` in file A is INT but DECIMAL in file B, DOUBLE is used in the returned column.
```

Comment:
We should give the specific union rules.
1. All INTEGERs will be unioned as the larger INTEGER.
2.An INTEGER column and a FLOAT column will be unioned as a DECIMAL column.
3. Others to STRING.